content/03.prior-knowledge.knowledge-graphs.md

```diff
@@ -1,17 +1,17 @@
-### Knowledge graphs for rare disease
+### Knowledge graphs 
+An intrinsic constraint in studying rare diseases is the lack of large, normalized datasets, which limits our ability to study key attributes of rare diseases. 
+A potentially powerful strategy for evaluating genotype-phenotype relationships or repurposing drugs when large datasets are scarce is to use knowledge graphs (KGs).
```

Comment:
I am of the opinion that using KGs as an acronym is going to be more trouble than it's worth – the sacrifice of readability will not be worth the words saved. (This is why I don't use LVs for latent variables in the transfer learning section.)

Reply:
thanks, i was borderline on the idea, so definitely happy to change this back!